### ISSUE TITLE
Mashling CREATE/BUILD - What is it doing to give these messages ?

### ISSUE THREAD

**Issue by ayh20** [TIBCOSoftware/mashling#47]
At the start of the process it throws some messages that can only come from it looking/processing all the files in my GOPATH ... it seems odd to say the least...

I know exactly why there are errors but why would Mashling to touching ANY of that code ?

PS D:\Data\Documents\Work\Content\Mashling\test3> mashling build
FATAL: command "fetch" failed: loadPackage("D:\\Data\\Documents\\GoWorkSpace\\src\\github.com\\ayh20\\GolangTraining\\02_Exercises", "github.com\\ayh20\\GolangTraining\\02_Exercises"): found packages main (main.go) and main2
 (main2.go) in D:\Data\Documents\GoWorkSpace\src\github.com\ayh20\GolangTraining\02_Exercises
FATAL: command "vendor" failed: exit status 1
Installing dependencies...
D:\Data\Documents\Work\Content\Mashling\test3

**Comment by pointlander** [comment]
Are you still having this issue after doing a 'go get -u github.com/TIBCOSoftware/mashling/...'?
If so, could you send me your recipe: asnodgra@tibco.com
Thanks @ayh20

**Comment by ayh20** [comment]
Still have the problem .... i'll contact you via your work email.

**Comment by mellistibco** [comment]
@pointlander please update this issue with your findings...

**Comment by pointlander** [comment]
I first attempted to replicate the issue by placing a .go file with an invalid import in my GOPATH and running 'mashling create app'. That didn't replicate the issue, so I added another .go file with a different package name in the same directory as the first .go file. I was then able to replicate the issue by running 'mashling create app'. It appears gb processes the entire GOPATH, and go doesn't allow multiple packages per directory: https://stackoverflow.com/questions/20427890/does-it-make-sense-to-have-two-packages-in-the-same-directory-golang
